charts/seed-monitoring/charts/grafana/templates/grafana-deployment.yaml

```
@@ -51,6 +51,8 @@ spec:
 {{- end }}
         - name: GF_AUTH_ANONYMOUS_ENABLED
           value: "true"
+        - name: GF_USERS_VIEWERS_CAN_EDIT
```

Should we allow end users to edit grafana dashboards too? This essentially means they get full access to prometheus. 

```suggestion
{{ if eq .Values.role "operators" }}
        - name: GF_USERS_VIEWERS_CAN_EDIT
```